core/src/main/java/io/questdb/cairo/pool/ReaderPool.java

```diff
@@ -66,26 +73,30 @@ public static class R extends TableReader implements PoolTenant<R> {
         private final ReaderListener readerListener;
         private Entry<R> entry;
         private AbstractMultiTenantPool<R> pool;
+        boolean active;
```

Comment:
This field seems to be unused. Do we need it?

Reply:
removed